core/src/kmx/kmx_plus.cpp

```diff
@@ -158,6 +159,24 @@ COMP_KMXPLUS_KEYS::valid(KMX_DWORD _kmn_unused(length)) const {
   return true;
 }
 
+std::u16string
+COMP_KMXPLUS_KEYS_ENTRY::get_string() const {
+  if (flags & LDML_KEYS_FLAGS_EXTEND) {
+    return std::u16string();  // error, extended
+  }
+  char16_t buf[2];
+  std::size_t len;
+  if (Uni_IsBMP(to)) {
+    buf[0] = (to & 0xFFFF);
```

Comment:
You've just added `Uni_IsBMP()` so let's go ahead and add a macro `Uni_UTF32BMPToUTF16`? or something like that? Just so we are consistent with our bit fiddling?

Reply:
done